print-ospf.c

```diff
@@ -836,6 +1099,20 @@ ospf_print_lsa(netdissect_options *ndo,
                         ND_PRINT("Capabilities: %s",
                                bittok2str(lsa_opaque_ri_tlv_cap_values, "Unknown", GET_BE_U_4(tptr)));
                         break;
+
+                    case LS_OPAQUE_RI_TLV_HOSTNAME:
+                        ND_PRINT("\n\t      Hostname: ");
+                        nd_printjnp(ndo, tptr, tlv_length);
+                        break;
+
+                    case LS_OPAQUE_RI_TLV_SID_LABEL_RANGE:
+                        ND_TCHECK_4(tptr);
+                        ND_PRINT("\n\t      Range size: %u", GET_BE_U_3(tptr));
+                        if (ospf_print_ri_lsa_sid_label_range_tlv(ndo, tptr+4, tlv_length-4) == -1) {
```

Comment:
Should be on 2 lines with indent.

Reply:
why is that ? - In line 1111 I am not violating any column boundary ?